examples/gocean/eg1/Makefile

```diff
@@ -36,6 +36,7 @@
 include ../../common.mk
 
 GENERATED_FILES += invoke_0_dag invoke_0_dag.png *.cl
+ENV = PSYCLONE_CONFIG=${PSYCLONE_DIR}/config/psyclone.cfg
```

Comment:
That's a nice solution. I was intending to re-write this example to get rid of the various `runme` scripts but this is a simpler solution for the short term :-)